types/decimal.go

```diff
@@ -474,6 +474,14 @@ func (d Dec) String() string {
 	return string(bzStr)
 }
 
+func (d Dec) Float64() float64 {
+	if value, err := strconv.ParseFloat(d.String(), 64); err != nil {
+		panic(err)
```

Comment:
This should return an error, not panic.

Reply:
Makes sense. I've changed the signature and also added the MustFloat64() that panics on error, what do you think?

Reply:
Sounds reasonable to me 👍